### PR TITLE
Support stderr and test output

### PR DESCRIPTION
**Pull request by darrenf**
tl;dr:
* add `STDERR` support to the notebook
* enable the inclusion of output from `Test.pm6`

Recent convert to both Jupyter and Perl 6, I observed that `STDERR` was ending up on the console rather than appearing in the web UI. Furthermore, when creating a notebook to demonstrate some `Test.pm6` behaviour _all_ of its output went to the console.

Some digging led me to discover that `$stderr` was not being initialised in `Result`. Adding that, and then making the kernel publish it, made `note` work as desired.

To make `Test.pm6` output appear it was necessary to reassign `$PROCESS::OUT` and `ERR` to the custom streams within the `Sandbox` - which themselves needed a `.say` method (as that's what `Test.pm6` calls, not `print`).

### CODE REVIEW

**Comment by bduggan** [comment]
Great, thank you!

(The travis test failures look unrelated).
